pkg/cloud/instance.go

```diff
@@ -237,7 +237,7 @@ func (c *client) GetOrCreateVMInstance(
 	p := c.cs.VirtualMachine.NewDeployVirtualMachineParams(offeringID, templateID, fd.Spec.Zone.ID)
 	p.SetNetworkids([]string{fd.Spec.Zone.Network.ID})
 	setIfNotEmpty(csMachine.Name, p.SetName)
-	setIfNotEmpty(csMachine.Name, p.SetDisplayname)
+	setIfNotEmpty(capiMachine.Name, p.SetDisplayname)
```

Comment:
This is not part of my requirements, but I think it is better to display a CAPI machine as display name in cloudstack UI. Otherwise, display name and name are identical in cloudstack UI, kind of redundancy.

Reply:
I think the question is do we think this changes adding value to our customers. If so, the confusing will go away when changes were rolled out. 

Once admin knows the changes, there is no confusing - just different. Unless customer has some scripts do some logic based on the display name.

Reply:
What the FS customer asked is CAPI machine name == k8s node name, and CS machine == k8s node name optionally. I think updating the VM display name is not necessary for now.